test/metabase/search/fulltext/scoring_test.clj

```diff
@@ -87,18 +100,18 @@
      {:model "card" :id 4 :name "order"}
      {:model "card" :id 5 :name "orders, invoices, other stuff", :description "a verbose description"}
      {:model "card" :id 6 :name "ordering"}]
-   ;; WARNING: this is likely to diverge between appdb types as we support more.
-    (testing "Preferences according to textual matches "
-     ;; Note that, ceteris paribus, the ordering in the database is currently stable - this might change!
-     ;; Due to stemming, we do not distinguish between exact matches and those that differ slightly.
+    ;; WARNING: this is likely to diverge between appdb types as we support more.
+    (testing "Preferences according to textual matches"
+      ;; Note that, ceteris paribus, the ordering in the database is currently stable - this might change!
+      ;; Due to stemming, we do not distinguish between exact matches and those that differ slightly.
       (is (= [["card" 1 "orders"]
               ["card" 4 "order"]
-             ;; We do not currently normalize the score based on the number of words in the vector / the coverage.
+              ;; We do not currently normalize the score based on the number of words in the vector / the coverage.
               ["card" 5 "orders, invoices, other stuff"]
               ["card" 6 "ordering"]
-             ;; If the match is only in a secondary field, it is less preferred.
+              ;; If the match is only in a secondary field, it is less preferred.
               ["card" 3 "classified"]]
-             (search :model "order"))))))
+             (search* :model "order"))))))
```

Comment:
Oops, I think the reason this needed a bypass is because it's testing the `:text` weight 🤦